demo/README-multitenant.md

```diff
@@ -6,25 +6,24 @@ To run either Alice or Faber with multitenancy enabled run with a `--multitenant
 ./run_demo faber --multitenant
 ```
 
-and/or:
+and:
 
 ```bash
 ./run_demo alice --multitenant
 ```
 
-This will enable a couple of additional options within the demo:
+This will enable a multitenancy support, and Faber and Alice will use a single Base agent.
 
-* `W` to create a new wallet (or switch to an existing wallet)
```

Comment:
Not sure why you removed these options?  I added them so you could test multi-wallet scenarios, e.g. faber issuing from one wallet and then verifying from another, alice receiving credentials in different wallets, etc.

Reply:
Oh, I'm sorry. I didn't think much about it. I restored these features.